hw-model/src/lib.rs

```diff
@@ -531,15 +543,31 @@ pub trait HwModel {
                 .write(|_| reg);
         }
 
-        // Set up the PAUSER as valid for the mailbox (using index 0)
-        self.soc_ifc()
-            .cptra_mbox_valid_pauser()
-            .at(0)
-            .write(|_| boot_params.valid_pauser);
-        self.soc_ifc()
-            .cptra_mbox_pauser_lock()
-            .at(0)
-            .write(|w| w.lock(true));
+        // Set up two valid PAUSERs  for the mailbox
+        #[cfg(not(feature = "fpga_realtime"))]
```

Comment:
Why not support multiple pausers for all implementations?

Reply:
Do we have any restrictions on the verilated model?

Reply:
I think this will work fine there too

Reply:
done